spec/abstract/ARM_HYP/ArchVSpace_A.thy

```diff
@@ -329,7 +329,7 @@ where
   "vgic_update vr f \<equiv> vcpu_update vr (\<lambda>vcpu. vcpu \<lparr> vcpu_vgic := f (vcpu_vgic vcpu) \<rparr> )"
 
 definition
-  vgic_update_lr :: "obj_ref \<Rightarrow> nat \<Rightarrow> ARM_A.virq \<Rightarrow> (unit,'z::state_ext) s_monad"
+  vgic_update_lr :: "obj_ref \<Rightarrow> nat \<Rightarrow> ARM_HYP_A.virq \<Rightarrow> (unit,'z::state_ext) s_monad"
```

Comment:
Not sure it's worth fixing, but it looks to me like this definition is in the Arch context, so should not need the name prefix.

Reply:
fixed

Reply:
well spotted!